### PR TITLE
Adjusting nested scopes examples in Routing Guide

### PR DESCRIPTION
**Pull request by willykaram**
Per issue #439, I've removed the nested scope from the example by modifying the code block and related prose mentioned.

I've also adjusted the prose earlier in the Guide that references a nested scope, which isn't as confusing as that mentioned by @chrismccord. That said, we could also potentially eliminate that example too. If so, LMK and I'll make that revision by removing the related prose/code block.

### CODE REVIEW

File: C_routing.md [hunk 1]
```diff
@@ -628,7 +628,7 @@ The `:browser` pipeline has five plugs: `plug :accepts, ["html"]` which defines
 
 Currently, the `:api` pipeline only defines `plug :accepts, ["json"]`.
 
-The router invokes a pipeline on a route defined within a scope. If no scope is defined, the router will invoke the pipeline on all the routes in the router. If we call `pipe_through/1` from within a nested scope, the router will invoke it on the inner scope only.
+The router invokes a pipeline on a route defined within a scope. If no scope is defined, the router will invoke the pipeline on all the routes in the router.
```
Comment (chrismccord) [inline C_routing.md]: Let's add this: "If we call `pipe_through` within a nested scope, the router will invoke all `pipe_through`'s from parent scopes, followed by the nested one. "

**Comment by chrismccord** [comment]
:ship: Thanks so much!

**Comment by willykaram** [comment]
Thanks for the feedback. I made a change to that effect and can revise further based on any additional feedback. I made the revision in a separate commit, but once things are finalized I can condenses them with an interactive rebase/squash if helpful.

**Comment by lancehalvorsen** [comment]
This looks great, @willykaram! Thanks so much. <3 <3 <3

Please squash the commits and we'll be ready to merge.

**Pull request closed without merging**